code/unit_tests/origins_tests.dm

```diff
@@ -0,0 +1,33 @@
+/datum/unit_test/cultures
+	name = "Cultures Test - All cultures shall be filled out"
+
+/datum/unit_test/cultures/start_test()
+	var/list/decl/origin_item/culture/all_cultures = decls_repository.get_decls_of_subtype(/decl/origin_item/culture)
+	for(var/decl/origin_item/culture/OC in all_cultures)
+		if(!istext(OC.name))
+			fail("Culture [OC.name] does not have a name!")
+		if(!istext(OC.desc))
+			fail("Culture [OC.name] does not have a description!")
+		if(!islist(OC.possible_origins))
+			fail("Culture [OC.name]'s possible_origins list is not a list!")
+		if(!length(OC.possible_origins))
+			fail("Culture [OC.name] does not have any possible origins!")
+	pass("All cultures are filled out properly.")
```

Comment:
Currently this is going to go...

'We failed'
'We failed'
'We passed'

Gotta put the pass after some sort of check that tells you nothing failed. Most unit tests do this with a var that increments when anything fails. 

Same for origins test.

Reply:
You would think fail() returns, but no

Reply:
Wait that makes no sense. I'm stupid